framework/src/base/Factory.C

```diff
@@ -92,7 +92,7 @@ Factory::getADValidParams(const std::string & obj_name)
 MooseObjectPtr
 Factory::create(const std::string & obj_name,
                 const std::string & name,
-                InputParameters & parameters,
+                const InputParameters & parameters,
```

Comment:
🙌🏼

Reply:
@loganharbour I bet you were surprised that I was _adding_ constness to MOOSE! ;-)

Reply:
I figured this was @rwcarlsen 😆